teaser/project.py

```diff
@@ -758,18 +759,98 @@ def export_aixlib(
                     )
 
         if report:
-            try:
-                from teaser.data.output.reports import model_report
-            except ImportError:
-                raise ImportError(
-                    "To create the model report, you have to install TEASER "
-                    "using the option report: pip install teaser[report] or install "
-                    "plotly manually."
-                )
-            report_path = os.path.join(path, "Resources", "ModelReport")
-            model_report.create_model_report(prj=self, path=report_path)
+            self._write_report(path)
+        return path
+
+    def export_besmod(
+        self,
+        internal_id=None,
+        examples=None,
+        path=None,
+        THydSup_nominal=None,
+        QBuiOld_flow_design=None,
+        THydSupOld_design=None,
+        custom_examples=None,
+        custom_script=None,
+        report=False
+    ):
+        """Exports buildings for BESMod simulation
+
+        Exports one (if internal_id is not None) or all buildings as
+        BESMod.Systems.Demand.Building.TEASERThermalZone models. Additionally,
+        BESMod.Examples can be specified and directly exported including the building.
+
+        Parameters
+        ----------
+
+        internal_id : float
+            setter of a specific building which will be exported, if None then
+            all buildings will be exported
+        examples: [string]
+            Names of BESMod examples to export alongside the building models.
+            Supported Examples are: "TEASERHeatLoadCalculation", "HeatPumpMonoenergetic" and "GasBoilerBuildingOnly".
+        THydSup_nominal : float or dict, optional
+            Nominal supply temperature(s) for the hydraulic system. Required for
```

Comment:
I thought this is a requirement? The rest is optional. Also, examples above should be optional, as well as internal_id, or?